test/metabase/lib/segment_test.cljc

```diff
@@ -100,7 +112,6 @@
                       :display-name      "PriceID-BBQ",
                       :long-display-name "PriceID-BBQ",
                       :effective-type    :type/Boolean,
-                      :description       "The ID is greater than 11 times the price and the name contains \"BBQ\".",
-                      :selected          true}]
+                      :description       "The ID is greater than 11 times the price and the name contains \"BBQ\"."}]
                     (map (partial lib/display-info query')
                          (lib/filters query'))))))))))
```

Comment:
The display-info for filter clauses has neither `:selected` nor `:filter-positions`. @kulyk Is this OK? (If I'm not mistaken, `:selected` was only set of segments, regular filters didn't have it.)

Reply:
Yes, I wouldn't expect `selected` flag on filter clauses